src/features/courses/course-controller.ts

```diff
@@ -64,6 +64,8 @@ import { getAverageGroupsBeforeDate, QUESTION_SQL_NAME, STUDENTTOPICOVERRIDE_SQL
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const Sequelize = require('sequelize');
 
+const ABSOLUTE_RENDERER_PATH_REGEX = /^(:?private\/|Contrib\/|webwork-open-problem-library\/|Library\/)/;
```

Comment:
nit: You could probably pull the `\/` out to the end of the regex.

Reply:
Probably, I pulled it from @doombot-exe 's regex on the renderer